app/src/main/res/layout-land/admin_pin_activity.xml

```diff
@@ -61,9 +61,9 @@
           android:id="@+id/input_pin"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_marginStart="140dp"
-          android:layout_marginTop="16dp"
-          android:layout_marginEnd="140dp"
+          android:layout_marginStart="168dp"
+          android:layout_marginTop="48dp"
+          android:layout_marginEnd="168dp"
```

Comment:
The left most border of input view is not aligned with that of `Save` Button where as in `ProfileResetPinActivity` `[landscape mode mock](https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/6f83fcd9-e72d-4941-a16f-6754a869e083/RG-MN-Register-Empty-37/)` the left side is aligned. So taking similar attributes from that mock, you can update the landscape mode mock for `AdminPinActivity`

Reply:
@rt4914 you mean to fix alignment of button in this PR?. As i mentioned we are fixing alignment as per ProfileInputView alignment updates in this PR.

Reply:
![Screenshot_1588685042](https://user-images.githubusercontent.com/9396084/81071313-29aafe00-8f02-11ea-85df-c531c39a4581.png)
![Screenshot_1588685058](https://user-images.githubusercontent.com/9396084/81071318-2c0d5800-8f02-11ea-8af5-4332b66f2c47.png)
![Screenshot_1588685172](https://user-images.githubusercontent.com/9396084/81071320-2ca5ee80-8f02-11ea-8457-580d49aec263.png)


Actually all these three button alignments are dependent on `ProfileInputView` because of this is margin is getting disturbed. Thats why it would nice to fix it in this PR itself as it is depended on `ProfileInputView`. If you still think that it should be separate fix, feel free to create an issue.

Reply:
Done.